packages/file-storage-common/src/auth.ts

```diff
@@ -67,6 +67,10 @@ export function getBlobHeaders(auth: IFileStorageAuthConfiguration): { [header:
     headers['Meeco-Subscription-Key'] = auth.subscription_key;
   }
 
+  if (auth.vault_access_token) {
+    headers['Authorization'] = auth.vault_access_token;
```

Comment:
Whenever I called the meecodevstorage service with this header it gives me 400 error. The only way it worked was if I did not include this header at all!

Reply:
what meeco dev storage service you are referring to?